packages/ckeditor5-dev-utils/package.json

```diff
@@ -14,14 +14,14 @@
     "fs-extra": "^8.1.0",
     "javascript-stringify": "^1.6.0",
     "pofile": "^1.0.9",
-    "postcss": "^7.0.17",
-    "postcss-import": "^12.0.0",
-    "postcss-loader": "^3.0.0",
-    "postcss-mixins": "^6.2.0",
-    "postcss-nesting": "^7.0.0",
+    "postcss": "^8.2.12",
+    "postcss-import": "^14.0.1",
+    "postcss-loader": "^4.2.0",
+    "postcss-mixins": "^7.0.3",
+    "postcss-nested": "^5.0.5",
```

Comment:
Think here https://github.com/postcss/postcss-nested is better to be used instead of to none official https://github.com/csstools/postcss-nesting